4001-5000/LIT4720Miracle.xml

```diff
@@ -50,21 +51,47 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         </profileDesc>
         <revisionDesc>
             <change who="DR" when="2017-10-09">Created entity</change>
+            <change when="2021-03-25" who="RL">Added bibliographical information, incipit, explicit</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
             <div type="bibliography">
                 <listBibl type="editions">
                     <bibl>
-                  <ptr target="bm:Grebaut1920MiraclesJesusII"/>
-                  <citedRange unit="page">827-831</citedRange>
+                  <ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">827-831</citedRange><citedRange unit="item">19</citedRange>
+                  <ptr target="bm:RompayArras1975Miracles"/><citedRange unit="item">19</citedRange>
                </bibl>
+               <listBibl type="translation">
+                 <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">827-831</citedRange></bibl>
+               </listBibl>
                 </listBibl>
                 <listRelation>
                <relation name="saws:formsPartOf" active="LIT4720Miracle" passive="LIT2382Taamme"/>
             </listRelation>
             </div>
+            <div type="edition">
+              <div type="textpart" subtype="incipit" xml:lang="gez">
+                <bibl><ptr target="bm:Grebaut1920MiraclesJesusII"/><citedRange unit="page">827-828</citedRange></bibl>
+                <ab>
+                  ወእንዘ፡ የኃልፍ፡ እግዚእ፡ ኢየሱስ፡ በሀገረ፡ ናይን፡ ወምስለ፡ አርዳኢሁ፡ ስም
```

Comment:
@PietroLiuzzo and @thea-m I have only put the line breaks in to make the transcription process easier to check. My understanding is that a line break on its own is not counted as a space. Is that correct? In the end once I have checked all the transcriptions once more then I could just remove the line breaks to avoid any confusion.

Reply:
I have now removed the line breaks